charts/generic-service/values.yaml

```diff
@@ -82,6 +82,33 @@ ingress:
        t:none,\
        setvar:tx.paranoia_level=2"
 
+# IP allowlist to be applied via ingress-nginx.
+allowlist: {}
+# Version of allowlist being applied (supplied by CI/CD)
+allowlist_version: ""
```

Comment:
is this just for auditing purposes?

Reply:
Yes, so we can track what version of the allowlist has been applied.